### ISSUE TITLE
Virtual Slide Component

### ISSUE THREAD

**Issue by jamesblasco** [surmon-china/vue-awesome-swiper#297]
I am wondering how to use a component in virtual slide because the array 
 virtual: {
    slides: ['Slide 1', 'Slide 2', 'Slide 3', 'Slide 4', 'Slide 5'],
  }, only accepts  html no vue componets

**Comment by surmon-china** [comment]
Yes, if you need components to render virtual slides, I think this business segment may not be reasonable.

**Comment by kid1412621** [comment]
@surmon-china May I ask why? I'm doing somthing sorta quiz test. I want to load quiz component into every virtual slider component,is that possible?or reasonable?

**Comment by siulca** [comment]
This answer is supposed to do what you and I want but I can't seem to get it working. 
https://stackoverflow.com/questions/51359121/how-can-i-insert-vue-component-into-virtual-slides

I've also created a codesandbox to show the problem: https://codesandbox.io/s/x28kxkxvnq

**Comment by Qingzii** [comment]
The virtual slideshow does not support vertical, which is a bit regretful
